go/obscuronode/host/host.go

```diff
@@ -170,7 +174,7 @@ func (a *Node) startProcessing() {
 			})
 
 		case <-a.exitNodeCh:
-			a.Enclave.Stop()
+			// a.Enclave.Stop()
 			return
```

Comment:
Should this be uncommented ?

Reply:
actually it should be removed, since we're handling te enclave lifecycle somewhere else now